code/src/pages/product-details/components/product-details-container/ProductDetailsContainer.scss

```diff
@@ -0,0 +1,79 @@
+@import "@design-system";
+
+.product-details {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: spacing(8) spacing(15);
+
+  &__image-wrapper {
+    flex: 3;
+    width: 100%;
+    min-width: 300px;
+    max-width: 700px;
+  }
+
+  &__image-wrapper img {
```

Comment:
define `img {}` styles inside `&__image-wrapper`

Reply:
Thanks for comment! I moved styles for image into previous style, now it looks better